application/auth/v1_controller.py

```diff
@@ -20,9 +20,9 @@
 auth_v1.url_prefix = "/v1/auth"
 
 
-@auth_v1.route('/register', methods=['POST'])
-@returns_json
-@json_convert(to=RegisterRequest)
+#@auth_v1.route('/register', methods=['POST'])
```

Comment:
you can just delete the thing that aren't used

Reply:
done